### PR TITLE
moved iana urn registration template as directed

### PR DESCRIPTION
**Pull request by bradleypeabody**
@kyzer-davis here's my attempt to address #23.  Some of the text I was able to just move over from what you had there but some fields didn't have a corresponding place in the new template so I just filled it out based on what made sense to me.

### CODE REVIEW

File: draft-ietf-uuidrev-rfc4122bis.md [hunk 1]
```diff
@@ -1512,8 +1518,79 @@ and feedback.
...
+UUID                   = time-low "-" time-mid "-"
+                         time-high-and-version "-"
+                         clock-seq-and-reserved
+                         clock-seq-low "-" node
+time-low               = 4hexOctet
+time-mid               = 2hexOctet
+time-high-and-version  = 2hexOctet
+clock-seq-and-reserved = hexOctet
+clock-seq-low          = hexOctet
+node                   = 6hexOctet
```
Comment (jimfenton) [inline draft-ietf-uuidrev-rfc4122bis.md:1563]:
```suggestion
node                   = 6*hexOctet
```

File: draft-ietf-uuidrev-rfc4122bis.md [hunk 2]
```diff
@@ -1512,8 +1518,79 @@ and feedback.
...
+  provided by the following ABNF {{RFC4234}}:
+
+~~~~ abnf
+UUID                   = time-low "-" time-mid "-"
+                         time-high-and-version "-"
+                         clock-seq-and-reserved
+                         clock-seq-low "-" node
+time-low               = 4hexOctet
+time-mid               = 2hexOctet
+time-high-and-version  = 2hexOctet
```
Comment (jimfenton) [inline draft-ietf-uuidrev-rfc4122bis.md:1560]:
```suggestion
time-high-and-version  = 2*hexOctet
```

File: draft-ietf-uuidrev-rfc4122bis.md [hunk 3]
```diff
@@ -1512,8 +1518,79 @@ and feedback.
...
+  The formal definition of the UUID string representation is
+  provided by the following ABNF {{RFC4234}}:
+
+~~~~ abnf
+UUID                   = time-low "-" time-mid "-"
+                         time-high-and-version "-"
+                         clock-seq-and-reserved
+                         clock-seq-low "-" node
+time-low               = 4hexOctet
+time-mid               = 2hexOctet
```
Comment (jimfenton) [inline draft-ietf-uuidrev-rfc4122bis.md:1559]:
```suggestion
time-mid               = 2*hexOctet
```

File: draft-ietf-uuidrev-rfc4122bis.md [hunk 4]
```diff
@@ -1512,8 +1518,79 @@ and feedback.
...
+
+  The formal definition of the UUID string representation is
+  provided by the following ABNF {{RFC4234}}:
+
+~~~~ abnf
+UUID                   = time-low "-" time-mid "-"
+                         time-high-and-version "-"
+                         clock-seq-and-reserved
+                         clock-seq-low "-" node
+time-low               = 4hexOctet
```
Comment (jimfenton) [inline draft-ietf-uuidrev-rfc4122bis.md:1558]:
```suggestion
time-low               = 4*hexOctet
```

**Review by jimfenton — COMMENTED** [review]
Corrections to ABNF (should be consistent with Section 4)

File: draft-ietf-uuidrev-rfc4122bis.md [hunk 5]
```diff
@@ -1512,8 +1518,79 @@ and feedback.
...
+  represent a UUID as a URN, it is necessary to convert the bit
+  sequence to a string representation.
+
+  Each field is treated as an integer and has its value printed as a
+  zero-filled hexadecimal digit string with the most significant
+  digit first.  The hexadecimal values "a" through "f" are output as
+  lower case characters and are case insensitive on input.
+
+  The formal definition of the UUID string representation is
+  provided by the following ABNF {{RFC4234}}:
```
Comment (mcr) [inline draft-ietf-uuidrev-rfc4122bis.md:1551]:
```suggestion
  provided by the following ABNF {{RFC5234}}:
```

Comment (mcr) [reply draft-ietf-uuidrev-rfc4122bis.md]: Maybe the ABNF could be referenced to this document, rather than repeated in the template.

**Review by mcr — CHANGES_REQUESTED** [review]

**Pull request merged**